ee/agent/reset.go

```diff
@@ -56,10 +56,16 @@ func DetectAndRemediateHardwareChange(ctx context.Context, k types.Knapsack) {
 	ctx, span := traces.StartSpan(ctx)
 	defer span.End()
 
-	k.Slogger().Log(ctx, slog.LevelDebug, "checking to see if database should be reset...")
-
 	serialChanged := false
 	hardwareUUIDChanged := false
+	munemoChanged := false
+
+	defer k.Slogger().Log(ctx, slog.LevelDebug, "checking to see if database should be reset...",
+		"serial", serialChanged,
+		"hardware_uuid", hardwareUUIDChanged,
+		"munemo", munemoChanged,
+	)
```

Comment:
This data also gets logged on ~85 if any of them are true -- want to remove that extra log since it'd be redundant with this one?

Reply:
I tweaked a little. How's this?

Reply:
Looks reasonable to me!

Reply:
Thummmbbbbss

Reply:
arrg. Committed too much. must revert

Reply:
Fixed.